scripts/compose-decision-prompt.py

```
@@ -702,15 +716,63 @@ def _test_pause_after_rollup_commit(home_fd: int) -> None:
+    maximum = _env_int("DECISION_ALERT_MAX_OUTPUT_BYTES", 65536, 1024, 262144)
     env = dict(os.environ)
     env["MISSION_CONTROL_HOME"] = home
-    proc = subprocess.run(
-        [decision_alert, *args], env=env, text=True, capture_output=True)
+
+    def cap_output() -> None:
+        resource.setrlimit(resource.RLIMIT_FSIZE, (maximum + 1, maximum + 1))
+
+    try:
+        with tempfile.TemporaryFile() as out, tempfile.TemporaryFile() as err:
+            proc = subprocess.Popen(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)